examples/a_treeple/treeple_tutorial_SA98.py

```diff
@@ -0,0 +1,115 @@
+"""
+=====================================
+Treeple tutorial for calculating S@98
```

Comment:
Same comment here regarding adding a paragraph describing what will be shown. 

Also spell out what S@98 is. Or to make this more general, state this is S@k-specificity and we will specifically look at S@98.